flocker/node/agents/test/test_blockdevice.py

```diff
@@ -1473,6 +1499,8 @@ def test_attach_volume_validate_size(self):
             new_volume.blockdevice_id, attach_to=self.this_node,
         )
 
+        self.assertNotEqual(attached_volume, None)
```

Comment:
I wouldn't bother. CPU cycles are not a bottleneck in our tests. In general better to just have one assertion that tests what we care about.